openaerostruct/examples/run_aerostruct_uCRM_multipoint.py

```diff
@@ -533,9 +536,6 @@
 # =======================================================================================
 # =======================================================================================
 
-from openmdao.utils.assert_utils import assert_rel_error
```

Comment:
How does this part of the code work? The example runscript is also used for unit testing? Is this being tested now?

Reply:
For some reason this example file includes a regression test in the latter part of the script, which was not actually tested during the testflo call. I removed this test because what it overwraps with other regression tests we already have.

Reply:
Yeah that sounds good, this seemed confusing from a user perspective.